views/about-page.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="/public/styles.css" rel="stylesheet" type="text/css" />
```

Comment:
you don't need the /public/ here, just the path after it
`<link href="/styles.css" rel="stylesheet" type="text/css" />`